package.json

```diff
@@ -370,7 +369,7 @@
                 },
                 {
                     "command": "vscode-docker.registries.azure.selectSubscriptions",
-                    "when": "view == dockerRegistries && viewItem == azureextensionui.azureSubscription",
+                    "when": "view == dockerRegistries && viewItem == azure;DockerV2;RegistryProvider;",
```

Comment:
Best explained via screenshot. There is a button to filter Azure subscriptions, so that the tree view only shows (and only loads) the Azure subscription(s) you care about. In the past, that filter button was shown next to each subscription item, which didn't make sense because the button is not "show only this subscription", but rather "choose which subscriptions to show":
![image](https://user-images.githubusercontent.com/36966225/77775632-33983200-7023-11ea-8ec2-ff9ead9912fb.png)
![image](https://user-images.githubusercontent.com/36966225/77775610-254a1600-7023-11ea-9312-189338519411.png)

Now, it's shown at the Azure Account root node:
![image](https://user-images.githubusercontent.com/36966225/77775749-5de9ef80-7023-11ea-8542-e972a9014604.png)

Reply:
cool